pages/edge/learn/instaboot.mdx

```diff
@@ -33,6 +33,11 @@ source: ...
 capabilities:
   # Enable InstaBoot:
   instaboot:
+    # You can optionally speciy a maximum age.
+    # If the maximum age is reached, a new snapshot will automatically
+    # be created, and the old ones discarded
+    max_age: 12h
+
```

Comment:
I think this is more confusing for newcomers. I don't see this setting should be here for the basic example.

Reply:
We should have a `### section` at the end of the article with the use case of the max_age in a bit more detail and how to customize

Reply:
Yeah that's valid, will do.